### ISSUE TITLE
Daemon fails to report server start due to JSON.parse Syntax Error (Modded Minecraft)

### ISSUE THREAD

**Issue by NopeDK** [pterodactyl/panel#1977]
**Background (please complete the following information):**
* Panel or Daemon: `Daemon`
* Version of Panel/Daemon: `0.7.17/0.6.13`
* Server's OS: `Debian 10.3`
* Your Computer's OS & Browser: `Windows 10 Build 18363, Chrome 81.0.4044.92 (Official Build) (64-bit)`

`Daemon and Panel is installed on the same server`

**Describe the bug**
Starting the server using the panel sometimes result in the panel being stuck in "Starting" as the daemon encounters a JSON parsing error and doesn't report a successful start.

When this occurs, the console in the panel stops auto-updating until you restart/stop/kill the instance. If you write a command in the console, nothing happens until you refresh the page and the console now shows everything that has happened since you last refreshed the page.

`php -v`: `PHP 7.3.14-1~deb10u1`
`uname -a`: `Linux Debian-103-buster-64-minimal 4.19.0-8-amd64 #1 SMP Debian 4.19.98-1 (2020-01-26) x86_64 GNU/Linux`
`docker info`:
```
Server Version: 19.03.8
Kernel Version: 4.19.0-8-amd64
Operating System: Debian GNU/Linux 10 (buster)
OSType: linux
Architecture: x86_64
CPUs: 16
Total Memory: 62.83GiB
```

**To Reproduce**
1. Restart a heavily modded server
2. If it gets stuck in starting, the error has occured
- Panel console no longer auto-updates, requires refresh
- See wings.log for error message

**Expected behavior**
- Server marked as started
- Daemon does not crash during JSON parse
- Panel console keeps auto-updating.

wings.log:
```
{"name":"wings","hostname":"xxxx","pid":##,"server":"xxxx","level":30,"msg":"Server status has been changed to STARTING","time":"2020-04-26T11:43:04.735Z","v":0}
{"name":"wings","hostname":"xxxx","pid":##,"level":60,"err":{"message":"Unexpected token l in JSON at position 151","name":"SyntaxError","stack":"SyntaxError: Unexpected token l in JSON at position 151\n    at JSON.parse (<anonymous>)\n    at Tail.<anonymous> (/srv/daemon/src/controllers/docker.js:248:40)\n    at Tail.emit (events.js:310:20)\n    at Tail.EventEmitter.emit (domain.js:482:12)\n    at ReadStream.<anonymous> (/srv/daemon/node_modules/tail/lib/tail.js:44:31)\n    at ReadStream.emit (events.js:310:20)\n    at ReadStream.EventEmitter.emit (domain.js:482:12)\n    at addChunk (_stream_readable.js:286:12)\n    at readableAddChunk (_stream_readable.js:264:11)\n    at ReadStream.Readable.push (_stream_readable.js:209:10)"},"msg":"A fatal error occured during an operation.","time":"2020-04-26T11:45:28.052Z","v":0}
```

latest.log (Minecraft):
```
[13:45:28] [Server thread/INFO] [net.minecraft.advancements.AdvancementList]: Loaded 1118 advancements
[13:45:28] [Server thread/INFO] [net.minecraft.advancements.AdvancementList]: Loaded 30 advancement trees
[13:45:28] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer]: Done (13.085s)! For help, type "help" or "?"
[13:45:28] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer]: Starting GS4 status listener
[13:45:28] [Query Listener #1/INFO] [net.minecraft.server.MinecraftServer]: Query running on x.x.x.x
```

**Comment by parkervcp** [comment]
I have a few heavily modded servers.

ATM 3 and D,D,S,S are just a few I have ran.

**Comment by NopeDK** [comment]
This is a DDSS 7.2b with added Sponge 7.1.8, LuckPerms and a few other permission/grief plugins and other niceties.

The tokens I encountered in error messages are also kinda weird (letter L and \\) which doesn't really appear in the log.

**Comment by DaneEveritt** [comment]
I think this is probably related to #1950

**Comment by NopeDK** [comment]
Sounds like it. Only difference I could see is that #1950 is the beta Wings whereas this is the old Daemon.

The many lines conjecture sounds reasonable as it usually during the last few seconds in the load process which outputs hundreds, if not thousands, of lines in a short span of time (enough to trigger the default DDOS throttling settings, I have disabled throttling while testing) (the many lines are due to achievements/advancements creating errors as recipes have been changed or something, with 1100 achievements and a stacktrace for each, it is probably 20k lines in less than 3 seconds).
